infoschema/metricschema/tables.go

```diff
@@ -71,3 +71,8 @@ func (vt *metricSchemaTable) Meta() *model.TableInfo {
 func (vt *metricSchemaTable) IterRecords(_ sessionctx.Context, _ kv.Key, _ []*table.Column, _ table.RecordIterFunc) error {
 	return nil
 }
+
+// Type implements table.Table Type interface.
+func (vt *metricSchemaTable) Type() table.Type {
```

Comment:
Prefer to move the `metricschema` subpackage to the `infoschema` package and derive `metricSchemaTable` from `infoschemaTable` to deduplicate some code.

Reply:
We can do this in the next PR, this pr was large enough.